src/main/java/com/codessquad/qna/web/domain/question/Question.java

```diff
@@ -1,20 +1,27 @@
-package com.codessquad.qna.web.model;
+package com.codessquad.qna.web.domain.question;
 
+import javax.persistence.*;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 
+@Entity
 public class Question {
     private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
 
-    private Integer id;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
 
+    @Column(nullable=false)
```

Comment:
애노테이션 필드에도 연산자 앞뒤로 한칸씩 공백을 잊지 말아주세요.
```suggestion
    @Column(nullable = false)

Reply:
앗..! 코드 포멧팅이 안되어있었네요..!🤦‍♀️ 짚어주셔서 감사합니다! 바로 수정하겠습니다!